docs/reference/seismogram.rst

```diff
@@ -29,6 +29,7 @@ Properties and Accessors
    Seismogram.events
```

Comment:
You also want to add a newsfragment (changelog).  
See https://github.com/epic-open-source/seismometer/blob/main/changelog/README.rst

Reply:
Added in 9fe042a